Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -52,6 +52,7 @@ void CLuaVehicleDefs::LoadFunctions()
         {"getVehicleNameFromModel", GetVehicleNameFromModel},
         {"getVehicleAdjustableProperty", GetVehicleAdjustableProperty},
         {"getHelicopterRotorSpeed", GetHelicopterRotorSpeed},
+        {"getVehicleRotorSpeed", GetVehicleRotorSpeed},
```

Comment:
New functions should use the new argument parser and not be defined in `CStaticFunctionDefinitions`. More context: https://github.com/multitheftauto/mtasa-blue/pull/1644#discussion_r480343689, https://github.com/multitheftauto/mtasa-blue/pull/1648